src/AppBundle/Entity/Notification.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Mgilet\NotificationBundle\Model\AbstractNotification;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="notification")
+ */
+class Notification extends AbstractNotification
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    protected $id;
+
+    /**
+     * @var User
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User", inversedBy="notifications")
+     */
+    protected $user;
+
+    /**
+     * @param string $subject
+     * @param string $message
+     */
+    public function __construct($subject, $message)
+    {
+        parent::__construct();
+        $this->setSubject($subject);
+        $this->setMessage($message);
+    }
+
+    /**
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * @return User
+     */
+    public function getUser()
+    {
+        return $this->user;
+    }
+
+    /**
+     * @param User $user
+     * @return Notification
+     */
+    public function setUser($user)
```

Comment:
What does this method do? Just setting up user or sending notification for user?

Method's name should be readable and easy to understand.